### PR TITLE
Rebranding Step-6

### PR DESCRIPTION
**Pull request by myang1010**
What (if any) features are you implementing?
- Updated step-6

Is there anything that you need from your teammate?
- This ticket doesn't change the "Add Another Expense" button style and background color. But add the "+" sign at the beginning of the button. We have another ticket to handle "Back" and "Continue" buttons, and "Add Another Expense" button can reuse the style from there.

Screenshots
- When "No" is selected by default.
>Screenshot 2023-12-06 at 9 09 58 AM

- When "Yes" is selected(Ignore the "Add Another Expense" button style for now. The "+" sign was added).
>Screenshot 2023-12-06 at 9 11 34 AM

- When there're multiple expensed added, the background color changes alternatively between light lavender and white(Still ignore the "Add Another Expense" button style).
>Screenshot 2023-12-06 at 9 12 10 AM

### CODE REVIEW

File: src/Components/ExpenseBlock/ExpenseQuestion.js [hunk 1]
```diff
@@ -171,7 +171,10 @@ const ExpenseQuestion = ({ expenseData, allExpensesData, setAllExpenses, deleteE
   );
 
   return (
-    <div key={index}>
+    <div
+      key={index}
+      style={{ backgroundColor: index >= 1 && index % 2 !== 0 ? 'var(--light-lavender-background)' : '' }}
```
Comment (CalebPena) [inline src/Components/ExpenseBlock/ExpenseQuestion.js:176]: What do you think about conditionally adding a class for background color. Also, I don't think that you need the `index >= 1 &&`.
```suggestion
      style={{ backgroundColor: index % 2 !== 0 ? 'var(--light-lavender-background)' : '' }}
```

File: src/Assets/styleController.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ export interface ITheme {
   terraCottaColor: string;
   midBlueColor: string;
   secondaryBackgroundColor: string;
+  lightLavenderBackground: string;
```
Comment (CalebPena) [inline src/Assets/styleController.ts:9]: Can you use `--secondary-background-color` from [Marianna/734/typography-updates](https://github.com/Gary-Community-Ventures/benefits-calculator/tree/Marianna/734/typography-updates)

**Review by CalebPena — CHANGES_REQUESTED** [review]

**Comment by sydney-devine** [comment]
@myang1010 for the alternating light lavender background, can we have it start with the very first expense? So if "yes", the first expense section should already have the lavender background, instead of on the 2nd expense added. ie: white background on main yes/no question; then: lavender, white, lavender, white, etc. 

When there is a lavender background, the input boxes should still be white:

image

For the "X" to close out an expense, I don't love that the button is right up against the border of the lavender. Can we give some padding above and to the right so that it floats inside the light lavender instead of cutting off the one corner of it?

File: src/Components/ExpenseBlock/ExpenseBlock.js [hunk 3]
```diff
@@ -90,7 +91,11 @@ const ExpenseBlock = ({ handleExpenseSourcesSubmit }) => {
   return (
     <>
       {createExpenseBlockQuestions()}
-      <Button variant="contained" onClick={(event) => handleAddAdditionalExpenseSource(event)}>
+      <Button
+        className="nav-btn back-button"
+        onClick={(event) => handleAddAdditionalExpenseSource(event)}
+        startIcon={<AddIcon sx={{ fontSize: 'medium !important', mr: '-8px' }} />}
```
[the remark this answers is not in the document]
Comment (myang1010) [reply src/Components/ExpenseBlock/ExpenseBlock.js:97]: Tried `inherit`, but the "+" looks pretty big to me. I guess the icon here is using different size unit? 

- current with `medium !important`
Screenshot 2023-12-06 at 12 50 40 PM

- switch to `inherit`
Screenshot 2023-12-06 at 12 51 18 PM

[the remark this answers is not in the document]
Comment (myang1010) [reply src/Components/ExpenseBlock/ExpenseBlock.js]: Have to keep `!important` here. But switch to `-0.5rem` for `mr`.

**Comment by myang1010** [comment]
@sydney-devine Here's the screenshot for the latest change,
Screenshot 2023-12-06 at 1 47 35 PM

Please let me know what you think. Thanks.

**Comment by sydney-devine** [comment]
> @sydney-devine Here's the screenshot for the latest change, Screenshot 2023-12-06 at 1 47 35 PM
> 
> Please let me know what you think. Thanks.

Thank you!!

Definitely step in the right direction. 

Hard to tell on my screen/from screenshot how much wiggle room we have on this, but any way to do a littl emore padding on the left with the lavender background? The first letter of the question is just hitting that edge and it looks a little awkward. 

image

In a similar vein, let's add a little more padding above the first question. However much we have underneath the input box for the amount will work on top too. Let's just make it symmetrical. 

image

It looks like the "X" is pushing down the question itself instead of top aligning with the expense type.

image

**Comment by myang1010** [comment]
@sydney-devine , here's the update screenshot based upon our last review feedback,

Changes made,
1. add extra padding to the top and bottom of the first question
2. add padding to the left of each question, so the first letters won't hit the edge
3. reduced the gap between "Close" button and question header
4. keep question input boxes white

Screenshot 2023-12-07 at 9 37 41 AM

Thanks

File: src/Assets/styleController.ts [hunk 4]
```diff
@@ -73,6 +73,7 @@ function generateMuiOverides(theme: ITheme) {
   const darkTerraCottaColor = theme.secondaryColor;
   const blackColor = '#2A2B2A';
   const midBlue = theme.midBlueColor;
+  const lavenderColor = '#ECDEED';
```
Comment (CalebPena) [inline src/Assets/styleController.ts:76]: Referenced this [here](https://github.com/Gary-Community-Ventures/benefits-calculator/pull/791/files/6ccd6ea24e623d2ec9ad78e1007c469333487b5d#r1418129878), but could we add this to the style controller with a name that describes what it does.

**Review by CalebPena — APPROVED** [review]
Good to merge if @sydney-devine  thinks that it looks good visually.

**Comment by sydney-devine** [comment]
> Good to merge if @sydney-devine thinks that it looks good visually.

Do we have an updated screenshot on this? I think we were going to make some updates so that it matches what Ivonne did to the income question

**Comment by myang1010** [comment]
> Do we have an updated screenshot on this? I think we were going to make some updates so that it matches what Ivonne did to the income question

@sydney-devine , here are the latest screenshots, 
step6-a


step6-b

**Comment by sydney-devine** [comment]
> > Do we have an updated screenshot on this? I think we were going to make some updates so that it matches what Ivonne did to the income question
> 
> @sydney-devine , here are the latest screenshots, step6-a
> 
> step6-b

Thanks @myang1010! Just one question, are you hovered over that Add another expense button in this screen?

**Comment by myang1010** [comment]
> Thanks @myang1010! Just one question, are you hovered over that Add another expense button in this screen?

Yes, just wanted to show that "+" button is there.

**Comment by sydney-devine** [comment]
> > Thanks @myang1010! Just one question, are you hovered over that Add another expense button in this screen?
> 
> Yes, just wanted to show that "+" button is there.

Thanks! Can I just snag a screen of it unhovered?

**Comment by myang1010** [comment]
> Thanks! Can I just snag a screen of it unhovered?

Screenshot 2023-12-13 at 1 27 46 PM

The PR for the back and continue buttons will set this `Add another expense button` and `Back` button to the same style.

**Comment by sydney-devine** [comment]
> > Thanks! Can I just snag a screen of it unhovered?
> 
> Screenshot 2023-12-13 at 1 27 46 PM
> The PR for the back and continue buttons will set this `Add another expense button` and `Back` button to the same style.

Got it - thanks! @CalebPena you can merge.

**Pull request merged**